src/test/scala/nl/knaw/dans/easy/multideposit/CustomMatchers.scala

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
```

Comment:
See comment by Linda on the DDM/EMD PR. This is the inception year, four digits, so shouldn't this be 2016?